Makefile

```diff
@@ -80,34 +80,53 @@ e2e-in-ci:
 	make create-tmp-env-ci && \
 	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml up e2e-tests
 
+reset-e2e-in-ci-with-build: reset-deps e2e-in-ci-with-build
+
 e2e-in-ci-with-build:
 	yarn build && \
-	docker compose -f docker-compose.yml up integration-deps -d && \
 	make create-tmp-env-ci && \
 	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml run --name e2e-tests e2e-tests make execute-e2e-from-within-container-cached || \
 	docker rm `docker ps -q -f status=exited`
 
 e2e-in-ci-cached:
-	docker compose -f docker-compose.yml up integration-deps -d && \
 	make create-tmp-env-ci && \
 	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml run --name e2e-tests e2e-tests make execute-e2e-from-within-container-cached || \
 	docker rm `docker ps -q -f status=exited`
 
+main-in-ci-cached:
+	make create-tmp-env-ci && \
+	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml run --name e2e-tests e2e-tests make start-main-ci || \
+	docker rm `docker ps -q -a -f status=exited`
+
+main-in-ci-cached-no-restart:
+	make create-tmp-env-ci && \
+	TMP_ENV_CI=tmp.env.ci docker compose run --name e2e-tests e2e-tests make start-main-ci || \
+	docker rm `docker ps -q -a -f status=exited`
+
+del-containers:
```

Comment:
don't want to add this as it is destructive to more than just the containers that get created in this repo.

Is the command you're looking for `docker compose rm -sfv`?